static/js/realm_linkifiers.js

```diff
@@ -0,0 +1,109 @@
+import marked from "../third/marked/lib/marked";
```

Comment:
I think we want to call this `linkifiers.js`; when we add stream-level linkifiers in the future, we'll likely want them to be managed by the same file.